src/LayerPlan.cpp

```diff
@@ -1029,7 +1051,8 @@ void LayerPlan::addWalls(const PathJunctions& walls, const Settings& settings, c
     }
 
     cura::Point p_end {0, 0};
-    order_optimizer.optimize();
+    const bool alternate_walls = settings.get<bool>("material_alternate_walls") && (layer_nr % 2 == (alternate_direction_modifier ? 1 : 0));
```

Comment:
To me the tertiary there is more clear than an xor. When I read `layer_nr % 2 == 0` it makes me read "on even layers" and `layer_nr % 2 == 1` it makes me read "on odd layers". The tertiary to switch between even and odd is pretty clear.

But I do agree with the rename of the variable, and it could also use some documentation regardless.

Implemented in 93eb68413031a84741a259cc29a809f7ec19ceda.